src/components/ChatApp/MessageComposer.react.js

```diff
@@ -396,7 +396,7 @@ class MessageComposer extends Component {
           />
         )}
         <TextAreaContainer
-          backgroundColor={textarea}
+          backgroundcolor={textarea}
```

Comment:
For resolving these warning, using `$` before the variable name is the convention. So, name it `$backgroundColor`

Reply:
Ok :+1: